data/transactions/logic/opcodes.go

```diff
@@ -298,6 +298,11 @@ var OpSpecs = []OpSpec{
 
 	// ABI support opcodes.
 	{0xb0, "log", opLog, asmDefault, disDefault, oneBytes, nil, 5, runModeApplication, opDefault},
+
+	// Dynamically indexing into LogicSigs
+	{0xc0, "txnas", opTxnas, assembleTxnas, disTxn, oneInt, oneAny, 5, modeAny, immediates("f")},
+	{0xc1, "gtxnas", opGtxnas, assembleGtxnas, disGtxn, oneInt, oneAny, 5, modeAny, immediates("t", "f")},
+	{0xc2, "gtxnsas", opGtxnsas, assembleGtxnsas, disTxn, twoInts, oneAny, 5, modeAny, immediates("f")},
```

Comment:
Is there room to put these near the other txn opcodes?

Reply:
It's a good thought, but there are only 2 slots left there, and need to make 3 more op codes, on top of the three here. I'm open to suggestions on tucking these 6 op codes somewhere else though.